src/App.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './app.css'
 
+
+const todos = []
 class App extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      todos: todos,
+      task: "", 
+      
+    }
+  }
+
+  addTask = e => {
+    e.preventDefault();
+    const newTask = {
+      task: this.state.task,
+      id: Date.now(),
+      completed: false,
+    }
+    
+    this.setState({
+      todos: [...this.state.todos, newTask],
+      task: "",
```

Comment:
This is nice. Good job resetting your input to an empty string. This is a better user experience.